docs/cloud-native-security/cspm-get-started-aws.asciidoc

```diff
@@ -59,21 +59,123 @@ image::images/cspm-cloudformation-template.png[The Add permissions screen in AWS
 
 When you return to {kib}, click *View assets* to review the data being collected by your new integration.
 
+
+[discrete]
+[[cspm-setup-organization-manual]]
+=== Manual authentication for organization-level onboarding
+
+NOTE: If you're onboarding a single account instead of an organization, skip this section.
+
+When using manual authentication to onboard at the organization level, you need to configure the necessary permissions using the AWS console for the organization where you want to deploy:
+
+* In the organization's management account (root account), create an IAM role called `cloudbeat-root` (the name is important). The role needs several policies:
+
+** The following inline policy:
+
+.Click to expand policy
+[%collapsible]
+====
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "organizations:List*",
+                "organizations:Describe*"
+            ],
+            "Resource": "*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "sts:AssumeRole"
+            ],
+            "Resource": "*",
+            "Effect": "Allow"
+        }
+    ]
+}
+```
+====
+
+** The following trust policy:
+
+.Click to expand policy
+[%collapsible]
+====
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::<Management Account ID>:root"
+            },
+            "Action": "sts:AssumeRole"
+        },
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "Service": "ec2.amazonaws.com"
+            },
+            "Action": "sts:AssumeRole"
+        }
+    ]
+}
+```
+====
+
+** The AWS-managed `SecurityAudit` policy.
+
+IMPORTANT: You must replace `<Management account ID>` in the trust policy with your AWS account ID.
+
+* Next, for each account you want to scan in the organization, create an IAM role named `cloudbeat-securityaudit` with the following policies:
+** The AWS-managed `SecurityAudit` policy.
+** The following trust policy:
+
+.Click to expand policy
+[%collapsible]
+====
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::<Management Account ID>:role/cloudbeat-root"
+            },
+            "Action": "sts:AssumeRole"
+        }
+    ]
+}
+```
+====
+
+IMPORTANT: You must replace `<Management account ID>` in the trust policy with your AWS account ID.
+
+After creating the necessary roles, authenticate using the <<cspm-use-instance-role, default instance role>> method.
```

Comment:
The user can authenticate using other methods as well, as long as aws credentials in the machine that cloudbeat runs are set up in a way that the `cloudbeat-root` can be assumed